.github/run-package-tests.sh

```diff
@@ -13,8 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# first argument can be a directory
+DIRS=$(find * -maxdepth 0 -type d -name '[A-Z]*')
+if [ "$#" -eq 1 ]; then
+    DIRS=$1
+elif [ "$#" -ne 0 ]; then
+    echo "usage: run-package-tests.sh [DIR]"
+    exit 1;
+fi
+
+# Use "composer-local.json" to avoid unwanted changes
+export COMPOSER=composer-local.json
+
 FAILED_FILE=$(mktemp -d)/failed
-for DIR in $(find * -maxdepth 0 -type d -name '[A-Z]*'); do {
+for DIR in ${DIRS}; do {
+    cp ${DIR}/composer.json ${DIR}/composer-local.json
+    # Update composer to use local packages
+    for i in bigquery,BigQuery core,Core logging,Logging, pubsub,PubSub storage,Storage; do
+        IFS=","; set -- $i;
+        if grep -q "\"google/cloud-$1\":" ${DIR}/composer.json; then
+            composer config repositories.$1 "{\"type\": \"path\", \"url\": \"../$2\", \"options\":{\"versions\":{\"google/cloud-$1\":\"1.100\"}}}" -d ${DIR}
+        fi
+    done
```

Comment:
We'll be sure to merge #5542 before merging this PR, so that these changes won't need to be a part of this PR.